game.shader.presets/addon.xml.in

```diff
@@ -10,10 +10,10 @@
     library_@PLATFORM@="@LIBRARY_FILENAME@"
     extensions="cgp|glslp|hlslp"/>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Shader Preset Support</summary>
-    <description lang="en">This library provides support for libretro "meta" shaders, or .cgp files.</description>
+    <summary lang="en_GB">Shader Preset Support</summary>
```

Comment:
Nexus brings language locale changes?

Reply:
No, "en" isn't a correct locale, the correct used by everything else is "en_GB". This change should have been made long ago, so I'm making it now and grouping it in with the actual Nexus updates.